src/content/configuration/other-options.md

```diff
@@ -48,7 +48,7 @@ As it happens, the AMD support in webpack ignores the defined name anyways.
 
 ## `bail`
 
-`boolean`
+`boolean: false`
```

Comment:
Please change according to https://webpack.js.org/contribute/writers-guide/#configuration-defaults-and-types
```suggestion
`boolean = false`
```

Reply:
Ok! I've updated with this change.

Separately, it might be a good idea to change the other default values in this document to match.

Reply:
@lencioni can you mention which ones? by looking at it on live i dont see any issues

Reply:
https://github.com/webpack/webpack.js.org/pull/3355